cmd/sentry/main.go

```diff
@@ -69,6 +68,11 @@ func main() {
 		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
 	}
 	flag.Parse()
+
+	if len(*depTA) > 0 {
+		log.Warn("--token-audience is deprecated and will be removed in v1.12")
```

Comment:
This is a matter of Dapr communicating with the control plane, so this change doesn't have an impact on users and doesn't require code changes from them. The only reason to support the deprecated option is so clusters can be upgraded doing a rollout and things will continue to work.

IMHO this doesn't need to adhere to the 2 releases policy since it's an internal thing.

Reply:
But the users have to make changes to the `--token-audience` flag right? At the end of the day it is a user facing change.

Reply:
The token-audience flag is normally set by Helm automatically with a default value, although users can override that (new thing added in 1.10)

But after this, the flag would become useless anyways.